packages/ketcher-polymer-editor-react/src/hooks/usePreview.ts

```diff
@@ -59,3 +59,30 @@ export const usePreview = (
 
```

Comment:
Do we still need usePreview ? If no please remove it and move calculatePreviewPosition to another file